Makefile

```diff
@@ -17,8 +17,20 @@ TOC_GENERATOR := $(CI_PATH)/gh-md-toc
 check-protoc:
 		./_tools/install-protoc-maybe.sh
 .PHONY: protogen
-protogen: check-protoc
-		$(GOCMD) get github.com/gogo/protobuf/protoc-gen-gogofaster
+protogen: check-protoc install-protoc-gen-gogofaster
 		./_tools/protogen_golang.sh
 		pip3 install --user grpcio_tools==1.13.0
 		./_tools/protogen_python.sh
+
+GOSRC ?= $(GOPATH)/src
+PROTOBUF_COMMIT_REF := 1.2.0
+GOGO_PROTOBUF_RELEASE_URL := https://github.com/gogo/protobuf/archive/v$(PROTOBUF_COMMIT_REF).zip
+PROTOC_GEN_GOGOFASTER_ZIP_PATH := $(CI_PATH)/protobuf-$(PROTOBUF_COMMIT_REF).zip
+install-protoc-gen-gogofaster: $(PROTOC_GEN_GOGOFASTER_ZIP_PATH)
```

Comment:
If you run it locally, and you have a different version of `protoc-gen-gogofaster` (not `v1.2.0`) it will generate the code with the wrong `protoc-gen-gogofaster`.

Reply:
true. Though I'm not sure it's a good idea to mess with user's globals.
up to you.

Reply:
Following the discussion out of this thread with @carlosms